.github/ISSUE_TEMPLATE/release.md

```diff
@@ -20,6 +20,7 @@ assignees: ''
 - [ ] Create release on GitHub
 - [ ] Update public docs
 - [ ] Announce release
+- [ ] Cherry-pick update of `ICINGA2_VERSION` and `CHANGELOG.md` into master
```

Comment:
> and CHANGELOG.md to keep it in sync.

Also, doing this manually feels like quite a downgrade from the previous automatism. I'd rather try to find a solution where we can keep this.

Reply:
I totally agree... with 1/2 of your opinion. Automatism is great. Depending on the frequency of the automated process. On its introduction that was like 10 times a day due to out of sync change logs. Now it would be once every several months.

Reply:
I am not sure what I am supposed to be reviewing here, but isn't https://github.com/Icinga/icinga2/pull/10039 the kind of automation what @julianbrost is asking for? If that PR gets accepted, I see no reason why we should also do this.

Reply:
There are two different things here:
* `ICINGA2_VERSION`: so far, this file isn't updated in the `master` branch which means `git describe` says something like `v2.14.0-238-ga8adfeda6`. In our package pipeline tooling, there's some special casing that fixes up the version number to something like `icinga2_2.14.2+238.ga8adfeda6`. This PR suggests updating that file on the master branch as well (which we don't do so far, [at the moment, it still says 2.14.0](https://github.com/Icinga/icinga2/blob/master/ICINGA2_VERSION)). #10039 looks like it does something similar to what our package tooling currently does but within our CMake config.
* `CHANGELOG.md`: so far, when a release is done on a `support/*` branch, Probot will create a PR like #9972 so that this version also appears in the same file on the `master` branch. This PR suggests to make this a manual step in the release workflow, that's what I was referring to with "downgrade from the previous automatism".